content/docs/rosa/ecr-secret-operator/_index.md

```diff
@@ -94,3 +99,46 @@ oc start-build ruby-sample-build --wait
 Build should succeed and push the image to the the private ECR Container repository
 
 ![Success Build](./images/build.png)
+
+
+### Create the ECR Secret Argo CD Helm Repo CRD
+
+* [OpenShift GitOps is installed](https://docs.openshift.com/container-platform/4.8/cicd/gitops/installing-openshift-gitops.html)
+* [Helm chart stored in ecr](https://docs.aws.amazon.com/AmazonECR/latest/userguide/push-oci-artifact.html)
+* aws ecr set-repository-policy --repository-name helm-test-chart --policy-text file:///tmp/repo_policy.json
```

Comment:
Can we delete this?  Instructions for this are in `iam_assume_role.md`, correct?

Reply:
Specifically, line 108 `aws ecr set-repository-policy...`